lib/plugins/aws/package/compile/events/apiGateway/lib/stage.js

```diff
@@ -50,53 +55,59 @@ module.exports = {
     // We'll change this in the future so that users can
     // opt-in for other features as well
     if (logs || TracingEnabled || Tags.length > 0) {
-      const stageResource = {
-        Type: 'AWS::ApiGateway::Stage',
-        Properties: {
-          DeploymentId: {
-            Ref: deploymentId,
-          },
-          RestApiId: this.provider.getApiGatewayRestApiId(),
-          StageName: this.provider.getStage(),
-          TracingEnabled,
-          Tags,
-        },
-      };
+      // --- currently commented out since this is done via the SDK in updateStage.js ---
```

Comment:
Why not remove that code?

Reply:
I was also thinking about removing this. However the overall goal is to get rid of the SDK-based solution ASAP.

Given that I decided to keep the code in there (commented out) so that we can easily bring it back later on without hunting through GitHub commits.

Futhermore others might see this and may wrap their head around solutions for that.

Reply:
Ok got it, so it's about CF way of handling this characteristics, which eventually may work someday.

Leaving this commented out, for informative reasons seems reasonable.